src/routes/admin/index.js

```diff
@@ -7,6 +8,12 @@ import userAdminAccessMiddleware from '../../middleware/userAdminAccessMiddlewar
 
 const router = express.Router();
 
+router.get('/requestErrors', getRequestErrors);
+router.get('/requestErrors/:id', getRequestError);
+router.delete('/requestErrors', deleteRequestErrors);
+
+// const router = express.Router();
+
 router.use(userAdminAccessMiddleware);
```

Comment:
I think I'd like to keep this line above all of the routes for clarity (or to avoid an authorization bug just in case the order here surprisingly does matter)